pkg/utils/gardener/secretsrotation/etcd_test.go

```diff
@@ -331,6 +331,35 @@ var _ = Describe("ETCD", func() {
 			))
 		})
 
+		It("should return the correct GVK list when not all resources are part from resourece list", func() {
```

Comment:
typo "resource"

Reply:
What exactly is the test case here? Is it when all the resources in the `resourcesToEncrypt` are not discovered? The string isn't very clear, can we improve it?